tests/cases/fourslash/todoComments20.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: dir1/node_modules/todoTest0.ts
+//// // TODO

```

Comment:
This last two tests creates some node_module files with Todos in them. 

Since no "spans" have been added to those files, the line of code verify.todoCommentsInCurrentFile(["TODO"]); pretty much checks for 0 todos (no more, no less). This makes the test scenario successful when the service don't report any todos when checking node_modules files even if they contain some.